open_spiel/python/pytorch/rcfr.py

```diff
@@ -0,0 +1,835 @@
+from __future__ import absolute_import
```

Comment:
At the very top of the file, san you add some text here describing RCFR? Does not have to be long (one or two sentences), but I would also like you to link to the original paper and Dustin's thesis. 

(You can just grab most of the text comment in the TF rcfr.py)